iothub/device/src/Transport/AmqpIoT/AmqpIoTConnector.cs

```diff
@@ -49,14 +49,9 @@ public async Task<AmqpIoTConnection> OpenConnectionAsync(TimeSpan timeout)
                 MaxFrameSize = AmqpConstants.DefaultMaxFrameSize,
                 ContainerId = CommonResources.GetNewStringGuid(),
                 HostName = _hostName,
+                IdleTimeOut = Convert.ToUInt32(_amqpTransportSettings.IdleTimeout.TotalMilliseconds),
```

Comment:
You lost the null check here. Was that not necessary?

Reply:
yeah, it wasn't required since a AmqpTransportSettings.IdleTimeout was TimeSpan and not TimeSpan? (nullable). 
These kinds of checks and validations are stronger in NET 5.0. Also, better support for co-orperative cancellation of a lot of async APIs :)